tests/core/models.py

```diff
@@ -24,10 +24,11 @@ class Article(models.Model):
     class Meta:
         app_label = 'core'
 
+
 class User(models.Model):
     name = models.TextField(db_index=True)
     user_id = models.TextField(blank=True, primary_key=True)
-    description = models.TextField(blank=True)
+    about = models.TextField(blank=True)
```

Comment:
`Article.description` uses a `snowball` analyzer whereas `User.description` is mapped to use an `edge_ngram_analyzer`, so ES complains that the mappings are different for fields with the same name.